pkg/piecestore/psserver/psdb/psdb.go

```diff
@@ -10,10 +10,11 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
-	"sync"
 	"time"
 
 	"github.com/gogo/protobuf/proto"
+	"github.com/lib/pq"
+	"github.com/mattn/go-sqlite3"
 	_ "github.com/mattn/go-sqlite3" // register sqlite to sql
 	"github.com/zeebo/errs"
 	"go.uber.org/zap"
```

Comment:
Most of the code in `psdb` will be replaced by code in `storagenode/storagenodedb`.